### PR TITLE
don't mirror binaries

### PR DESCRIPTION
**Pull request by traian-schiau**
Also drop cross compiling, (incompatible with cgo)

### CODE REVIEW

File: crypto/signing/mcl/g1_test.go [hunk 1]
```diff
@@ -3,7 +3,8 @@ package mcl
 import (
 	"testing"
 
-	"github.com/ElrondNetwork/elrond-go/crypto/signing/mcl/bls-go-binary/bls"
+	"github.com/herumi/bls-go-binary/bls"
+
```
Comment (AdoAdoAdo) [inline crypto/signing/mcl/g1_test.go]: remove empty line

File: crypto/signing/mcl/suiteBLS12_381.go [hunk 2]
```diff
@@ -20,13 +20,13 @@ type SuiteBLS12 struct {
...
 
-// Enable if compiling mcl/bls/bls-go-binary with BLS_SWAP_G flag
+// Is enabled if compiling mcl/bls/bls-go-binary with BLS_SWAP_G flag
 // Compiling with the flag will give Public Keys on G1 (48 bytes) and Signatures on G2 (96 bytes)
 // Compiling without the flag will give Public Keys on G2 and Signatures on G1
 // For Elrond the public keys for the validators are known during an epoch and also are not set on blocks
 // BLS signatures are however set on every block header, so in order to optimise the header size flag will be false
 // to have smaller signatures, so on G1(48 bytes)
-const blsSwapG = false
+var bls_swap_g = bls.IsSwapG()
```
Comment (AdoAdoAdo) [inline crypto/signing/mcl/suiteBLS12_381.go]: please rename to blsSwapG

Comment (traian-schiau) [reply crypto/signing/mcl/suiteBLS12_381.go]: done

File: crypto/signing/mcl/g1_test.go (hunk 1 above)
Comment (traian-schiau) [reply crypto/signing/mcl/g1_test.go]: done

File: crypto/signing/mcl/gt_test.go [hunk 3]
```diff
@@ -3,7 +3,8 @@ package mcl
 import (
 	"testing"
 
-	"github.com/ElrondNetwork/elrond-go/crypto/signing/mcl/bls-go-binary/bls"
+	"github.com/herumi/bls-go-binary/bls"
+
```
Comment (ccorcoveanu) [inline crypto/signing/mcl/gt_test.go]: Remove empty line

File: go.mod [hunk 4]
```diff
@@ -2,6 +2,8 @@ module github.com/ElrondNetwork/elrond-go
 
 go 1.13
 
+replace github.com/herumi/bls-go-binary => github.com/AdoAdoAdo/bls-go-binary v0.0.0-20200313082405-cbbb22d84df3
```
Comment (iulianpascalau) [inline go.mod]: What is this?

Comment (traian-schiau) [reply go.mod]: To use a fork for now.

Comment (AdoAdoAdo) [reply go.mod]: this is until pull request is accepted in herumi/bls-go-binary

File: crypto/signing/mcl/gt_test.go (hunk 3 above)
Comment (traian-schiau) [reply crypto/signing/mcl/gt_test.go]: done

File: go.mod (hunk 4 above)
Comment (iulianpascalau) [reply go.mod]: But why the fork was not done on ElrondNetwork?

Comment (AdoAdoAdo) [reply go.mod]: it is only temporary.
It had the changes I did originally for upstream, AdoAdoAdo is my account.

Comment (traian-schiau) [reply go.mod]: changed to github.com/ElrondNetwork/bls-go-binary v0.0.0-20200313082405-cbbb22d84df3

**Pull request merged**